pkg/kube/object.go

```diff
@@ -25,13 +25,13 @@ const (
 	KindNode Kind = "Node"
 
 	KindPod                   Kind = "Pod"
-	KindReplicaSet            Kind = "ReplicaSet"
+	KindReplicaSet            Kind = "ReplicaSet.apps"
```

Comment:
As we discussed we cannot mix Kubernetes Kinds with fully qualified resource names.